source/grid/grid_in.cc

```diff
@@ -181,15 +181,15 @@ GridIn<dim, spacedim>::read_vtk(std::istream &in)
   unsigned int n_ints;
 
   if (keyword == "CELLS")
-    {
-      in >> n_geometric_objects;
-      in >> n_ints; // Ignore this, since we don't need it.
+  {
+    in >> n_geometric_objects;
+    in >> n_ints; // Ignore this, since we don't need it.
```

Comment:
There are a lot of whitespace changes in this patch that make it really difficult to review. Please run `make indent` in your build directory to normalize whitespace. This is going to make it substantially easier to review what *really* is different.